addons/sale_mrp/tests/test_sale_mrp_flow.py

```diff
@@ -556,9 +554,6 @@ def test_01_sale_mrp_delivery_kit(self):
         self.assertEqual(so.invoice_status, 'to invoice', 'Sale MRP: so invoice_status should be "to invoice" after complete delivery of a kit')
 
     def test_02_sale_mrp_anglo_saxon(self):
-        self._test_sale_mrp_anglo_saxon('product')
-
-    def _test_sale_mrp_anglo_saxon(self, finished_product_type):
```

Comment:
You add it in the previous commit to remove it just after ?

Reply:
I added the test in the first commit (_[FIX] stock_account,sale_mrp: consider consu kit for COGS_) so that the latter is fully independent. I think this is better (even if I create and delete the test in the same PR) in the case this specific commit is reuse on another branch, or in the case the second commit is reverted. What do you think ?

Reply:
I am not opposite to have fully independent commit.
Then it seems that you don't test the anglo_saxon when kit is storable, because we only force **new** product/kit, we should test also with storable kit to be sure to not break old storable kit accounting.